src/Sulu/Bundle/ContentBundle/Search/Metadata/StructureDriver.php

```diff
@@ -150,9 +151,16 @@ public function loadMetadataForClass(\ReflectionClass $class)
         // index the webspace
         $indexMeta->addFieldMapping('webspace_key', array(
             'type' => 'string',
+            'index_strategy' => Field::INDEX_UNSTORED,
             'field' => $this->factory->createMetadataProperty('webspaceKey'),
         ));
 
+        $indexMeta->addFieldMapping('state', array(
+            'type' => 'string',
+            'index_strategy' => Field::INDEX_UNSTORED,
```

Comment:
We index them, but their values are not retrievable. (Zend Lucene has `Text` (stored + indexed) `unStored` (indexed but not stored), `unIndexed` (stored but not indexed))